.github/workflows/code-quality.yml

```
@@ -0,0 +1,80 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)